oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleSourceSchemaReader.java

```diff
@@ -56,6 +56,17 @@ public class OracleSourceSchemaReader extends CommonSchemaReader {
     Types.DECIMAL
   );
 
+  private String sessionID;
+
+  public OracleSourceSchemaReader() {
+    super();
```

Comment:
nit: call the other constructor `OracleSourceSchemaReader(null)`

This is functionally the same today, but this is slightly better in case the other constructor is later modified to have any other logic.

Reply:
Good point -- and then the `sessionID` variable can be `final` as well. Done1